src/components/BreedsList.vue

```diff
@@ -186,7 +187,7 @@ export default {
       this.indexClicked++;
       breed.isActive = !breed.isActive;
       if (breed.isActive) {
-        this.lastBreedState.push(breed);
+        this.lastBreedState.push(id);
       }
```

Comment:
Sedaj sem namesto breed pushov samo id, torej so v lastBreedState, samo id od klikanih kartic in ne celoten breed.